server/src/main/java/io/spine/server/event/EventBus.java

```diff
@@ -177,17 +177,39 @@ public EventStore getEventStore() {
     /**
      * Validates and posts the event for handling.
      *
-     * <p>Does performs the same as the
+     * <p>Performs the same action as the
      * {@linkplain CommandOutputBus#post(Message, StreamObserver)} parent method}, but does not
      * require any response observer.
      *
      * <p>This method should be used if the callee does not care about the event acknowledgement.
      *
+     * <p>Use the {@code Bus} class abstract methods to modify the behavior of posting.
+     *
      * @param event the event to be handled
      * @see CommandOutputBus#post(Message, StreamObserver)
      */
-    public void post(Event event) {
-        post(event, StreamObservers.<Response>noOpObserver());
+    public final void post(Event event) {
+        post(event, StreamObservers.<Event>noOpObserver());
+    }
+
+    /**
+     * Validates and posts the event for handling.
+     *
+     * <p>Performs the same action as the
+     * {@linkplain CommandOutputBus#post(Iterable, StreamObserver)} parent method}, but does not
+     * require any response observer.
+     *
+     * <p>This method should be used if the callee does not care about the events acknowledgement.
+     *
+     * <p>Use the {@code Bus} class abstract methods to modify the behavior of posting.
+     *
+     * <p>This method defines the general posting flow and should not be overridden.
+     *
+     * @param events the events to be handled
+     * @see CommandOutputBus#post(Message, StreamObserver)
+     */
+    public void post(Iterable<Event> events) {
```

Comment:
Can't we make it `final` as well? What if we try to rewrite the Mockito tests? 

I am asking to tweak the tests for several reasons:

* it should be `final` by design;
* we could avoid `<p>Use the {@code Bus} class abstract methods to modify the behavior of posting` and `<p>This method defines the general posting flow and should not be overridden.`.